pkgs/applications/misc/gkrellm/default.nix

```diff
@@ -30,6 +31,23 @@ stdenv.mkDerivation rec {
   makeFlags = [ "STRIP=-s" ];
   installFlags = [ "DESTDIR=$(out)" ];
 
+  # This icon is used by the desktop file.
+  postInstall = ''
+    install -Dm444 -T src/icon.xpm $out/share/pixmaps/gkrellm.xpm
+  '';
+
+  desktopItems = [
+    (makeDesktopItem {
+      name = "gkrellm";
+      exec = "gkrellm";
+      icon = "gkrellm";
+      desktopName = "GKrellM";
+      genericName = "System monitor";
+      comment = "The GNU Krell Monitors";
+      categories = "System;Monitor;";
+    })
+  ];
+
   meta = {
     description = "Themeable process stack of system monitors";
     longDescription = ''
```

Comment:
Last question: Would you mind to add yourself as a maintainer? If not we can merge it.

Reply:
Sure, I'd be happy to do that.  I can't make the changes right this minute, but I'll do that and squash this down to a single commit.